tests/cli/test_cli_classification_mol.py

```diff
@@ -145,7 +145,7 @@ def test_predict_output_structure(monkeypatch, data_path, model_path, tmp_path):
         m.setattr("sys.argv", args)
         main()
 
-    assert (tmp_path / "preds_0.csv").exists()
+    assert (tmp_path / "preds.csv").exists()
```

Comment:
Maybe also test that the individual file is saved?

Reply:
That is a good suggestion. I have added some tests to ensure the output file structure.